src/fdb5/database/ArchiveVisitor.h

```diff
@@ -30,11 +30,11 @@ class ArchiveVisitor : public BaseArchiveVisitor {
 
 public: // methods
 
-    ArchiveVisitor(Archiver &owner, const Key &field, const void *data, size_t size, const ArchiveCallback& callback = CALLBACK_NOOP);
+    ArchiveVisitor(Archiver& owner, const Key& field, const void* data, size_t size, const ArchiveCallback& callback = CALLBACK_NOOP);
```

Comment:
missing include since it's defaulting to noop; otherwise missing forward decl
`ArchiveCallback`

not in this PR but I see a lot of `namespace metkit { class MarsRequest; }` where `mars` namespace is missing